test/SymbolGraph/Module/ExportedImport.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %S/Inputs/ExportedImport/A.swift -module-name A -emit-module -emit-module-path %t/A.swiftmodule
+// RUN: %target-swift-frontend %s -module-name ExportedImport -emit-module -emit-module-path /dev/null -I %t -emit-symbol-graph -emit-symbol-graph-dir %t/
+// RUN: %FileCheck %s --input-file %t/ExportedImport.symbols.json
+
+@_exported import A
+
+// CHECK: "precise":"s:1A11SymbolFromAV"
```

Comment:
I think it might also be worth adding a test that makes sure that `ExportedImport@A.symbols.json` or the like doesn't get generated as a result of the module-inclusion rules getting relaxed. Something like:

```
// RUN: ls %t | %FileCheck --check-prefix FILES

...

// FILES-NOT: ExportedImport@A.symbols.json
```

Again, mainly just me being overly paranoid here. 😅